dist/cms.js

```diff
@@ -151,12 +151,45 @@ function getWordPressContentType() {
   return content;
 }
 
+/**
+ * Obtains data about Interactivity API elements on the page..
+ *
+ * @returns {object} Object with fields `total` and `total_by_type`.
+ */
+function getInteractivityAPIRegionCounts() {
+  // Look for data-wp-interactive elements.
+  const interactiveRegions = document.querySelectorAll('[data-wp-interactive]');
+
+  // Count the regions by type.
+  const regionTypeCounts = [];
+  interactiveRegions.forEach( region => {
+    const type = region.getAttribute('data-wp-interactive');
+    regionTypeCounts[ type ] = ( regionTypeCounts[ type ] || 0 ) + 1;
+  })
+
+  return {
+    total: interactiveRegions.length,
+    total_by_type: regionTypeCounts,
+  };
+}
+
+/**
+ * Check if the page uses the interactivity API.
+ *
+ * @returns {boolean} Whether any interactivity API regions are present.
+ */
+function usesInteractivityAPI() {
+  return !!document.querySelector('[data-wp-interactive]');
+}
+
 const wordpress = {
   block_theme: usesBlockTheme(),
   has_embed_block: hasWordPressEmbedBlock(),
   embed_block_count: getWordPressEmbedBlockCounts(),
   scripts: getWordPressScripts(),
   content_type: getWordPressContentType(),
+  uses_interactivity_api: usesInteractivityAPI(),
+  interactivity_api_region_count: getInteractivityAPIRegionCounts(),
```

Comment:
I think I prefer your original approach of putting an object in here that is extendable with more data in the future, to avoid potentially having lots of Interactivity API top-level properties in here in the future.

I think `uses_interactivity_api` in here on its own makes sense, but for anything more granular about the Interactivity API maybe we can use an object like `interactivity_api_usage`? For now it could have things like `regionCount` etc., and the list could be expanded as needed in the future.

Reply:
> maybe we can use an object like interactivity_api_usage? For now it could have things like regionCount etc., and the list could be expanded as needed in the future

Yes, good suggestion, I will update the naming here.

Reply:
updated to `interactivity_api_usage `, also renamed return variable names a bit